enhancements/suspend-and-resume-scan-schedule.md

```diff
@@ -0,0 +1,231 @@
+  information.  Can we do this?
+
+### Test Plan
+
+This feature can be easily tested using our existing end-to-end test suite. We
+don't necessarily need to rely on testing the functionality of `CronJob`, but
+we can certainly test that the Compliance Operator updates the `suspend`
+attribute according to the annotation.
+
+We should have an end-to-end test that performs the following:
+
+1. Creates a new `ComplianceScan`, where the actual scan schedule doesn't necessarily matter
+2. Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=false`
+3. Pause the `ComplianceScan` using an annotation
+4. Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=true`
+5. Resume the `ComplianceScan` by removing the annotation
+6. Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=false`
+
+### Upgrade / Downgrade Strategy
+
+Considering the Compliance Operator has typically followed a rolling release
+strategy, where fixes and features are never backported, that same concept will
+apply to this features.
+
+Attempting to suspend a scan schedule on an older release will simply not work.
+Suspending a scan schedule and downgrading the operator will result in the same
+behavior, where the older operator will not support suspending or resuming a
+scan.
+
+In the case where a cluster administrator suspends a scan, then downgrades the
+operator, the `CronJob` associated with the scan rerunner will already be in a
+suspended state. This will remain true even after the Compliance Operator is
+downgraded. However, a cluster administrator will not be able to resume a scan
+from an older version of the Compliance Operator that doesn't support the
+feature. Cluster administrators will need to modify the `CronJob.suspend`
+property directly to resume the scan schedule.
+
+### Version Skew Strategy
+
+This feature relies on `CronJob` semantics, which are currently considered a
+stable feature of Kubernetes.
+
+### Operational Aspects of API Extensions
+
+The life cycle of the `ComplianceScan` or its underlying `CronJob` should not
+be impacted by this feature. Cluster administrators will be able to clean up
+`ComplianceScan` if they are suspended or not.
+
+#### Failure Modes
+
+- Describe the possible failure modes of the API extensions.
+- Describe how a failure or behaviour of the extension will impact the overall cluster health
+  (e.g. which kube-controller-manager functionality will stop working), especially regarding
+  stability, availability, performance and security.
+- Describe which OCP teams are likely to be called upon in case of escalation with one of the failure modes
+  and add them as reviewers to this enhancement.
+
+#### Support Procedures
+
+Logs will emit information loggingwhen scan schedules are suspended and
+resumed. Knowing when a scan schedule is suspended or resumed is important from
+a compliance perspective, and warrants a higher log level than debug.
```

Comment:
> Knowing when a scan schedule is suspended or resumed is important from
a compliance perspective, and warrants a higher log level than debug.

Similarly, is there value in logging that a scan was not started on schedule because it was suspended?

Reply:
That's a good question, but it might mean we'd need to dig into the implementation details of the `CronJob`.